### ISSUE TITLE
Supplies in ORvR frequently unable to be picked up

### ISSUE THREAD

**Issue by Rydiak** [WarEmu/WarBugs#17701]
Supply boxes from BOs and players are frequently becoming bugged since patch, where they are unable to be picked up with 1 of 2 errors:

1) Removed bugged capturer [name] from Order/Destruction Supplies.
2) [Name] is already interacting with this object.

In both cases, no one is actually interacting with the supplies.

**Comment by MaxHayman** [comment]
Can you please make note of the location. The bug tracket template clearly states "Testing Screenshots/Videos/Evidences (always needed)"

**Comment by Rydiak** [comment]
I have noticed it in High Pass and in Dragonwake last night. High Pass was from supply drops via player kills, while Dragonwake was both from BOs and player kills. I'll see if I have a screenshot of the error messages.

**Comment by 5119ror** [comment]
I experienced this bug this morning. I had successfully picked up a supply from TH in Thunder Mountain, then got into a scrap and dropped the box during battle. After combat was over, I tried to pick my box back up and was told "[my own character name] is already interacting with this object." I tried repeatedly to pick it up, moving around and repositioning (etc) to try and "reset" whatever bugged interaction might be happening, but the only message I ever got when clicking the box was the "already interacting" error.
